internal/app/machined/pkg/controllers/block/disks.go

```diff
@@ -156,7 +157,9 @@ func (ctrl *DisksController) analyzeBlockDevice(ctx context.Context, r controlle
 	touchedDisks[device.Metadata().ID()] = struct{}{}
 
 	return safe.WriterModify(ctx, r, block.NewDisk(block.NamespaceName, device.Metadata().ID()), func(d *block.Disk) error {
+		d.TypedSpec().DevPath = filepath.Join("/dev", device.Metadata().ID())
 		d.TypedSpec().Size = size
+		d.TypedSpec().HumanSize = humanize.Bytes(size)
```

Comment:
`HumanSize` feels weird, maybe `PrettySize`? :thinking:

Reply:
yes... idk. I was thinking of the `-H` option

Reply:
renamed to `PrettySize`